Tetris Clone C++/main.cpp

```diff
@@ -87,10 +88,14 @@ int main(int argc, char const *argv[]) {
                         canInstantDrop = false;
                     }
                     else if (event.key.code == sf::Keyboard::D) {
-                        piece.rotateClockwise();
+                        if (piece.rotateClockwise()) {
+
```

Comment:
no this was legacy if it returned a tuple we could use it to reset the frame counter.